library/career-post.html

```diff
@@ -127,4 +127,4 @@
 </div>
 
 
-${require('./layouts/footer.html')}
+${require('./layouts/footer-blog.html')}
```

Comment:
this page is not related to blog
I don't like the name of the footer.
maybe it's a footer-light ? or footer-simple ?
what is different with standard footer ?